src/sentry/processing/backpressure/memory.py

```diff
@@ -4,7 +4,7 @@
 
 import rb
 import requests
-from rediscluster import RedisCluster
+from redis import RedisCluster  # type: ignore[attr-defined]
```

Comment:
these should be absolute imports rather than type ignored -- it's just that they aren't exported in `redis/__init__.py`